web/src/lib/utils/markdownReader.ts

```diff
@@ -62,4 +63,5 @@ export type TaskWithoutLinks = {
   callToActionText: string;
   postOnboardingQuestion: string;
   contentMd: string;
+  required?: boolean;
```

Comment:
do opportunities have this required tag also? I thought this was just for tasks

Reply:
Just for tasks I think, is the `TaskWithoutLinks` type here also being used for opportunities?

Reply:
oops. the way github rendered the diff made me misread this one. all good